src/assessments/seo/MetaDescriptionKeywordAssessment.js

```diff
@@ -64,14 +65,32 @@ class MetaDescriptionKeywordAssessment extends Assessment {
 	 * @returns {Object} Result object with score and text.
 	 */
 	calculateResult( i18n ) {
-		if ( this._keywordMatches >= this._config.parameters.recommendedMinimum  ) {
+		// GOOD result when at least one sentence contains every keyword term at least once or twice.
+		if ( this._keywordMatches.perSentence.some( sentence => Math.min( ...sentence ) >= 1 && Math.min( ...sentence ) <= 2 ) ) {
```

Comment:
Apparently, the GOOD condition was formulated ambiguously.
`GOOD if all content words from the keyphrase (or synonym phrase) are matched within one sentence in the metadescription once or twice.` 
should be read as
`There exists a sentence or two sentences, which contain all content words from the keyphrase or a synonym phrase.`